packages/demo-app-ts/src/App.tsx

```diff
@@ -167,8 +168,10 @@ class App extends React.Component<{}, AppState> {
               <BarsIcon />
             </PageToggleButton>
           </MastheadToggle>
-          <MastheadBrand>
-            <Brand src={imgBrand} alt="Patternfly Logo" heights={{ default: '40px' }} />
+          <MastheadBrand data-codemods>
```

Comment:
Is this to be expected or can this be removed from here and `MastheadLogo`?
```suggestion
          <MastheadBrand>
```

Reply:
good catch - i merged your suggestion!